src/app/tests/suites/certification/Test_TC_DRLK_2_13.yaml

```
@@ -0,0 +1,708 @@
+      command: "ClearAliroReaderConfig"
+
+    - label: "Step 8: TH reads AliroReaderVerificationKey attribute from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.A0080
+      command: "readAttribute"
+      attribute: "AliroReaderVerificationKey"
+      response:
+          value: null
+
+    - label: "Step 9: TH reads AliroReaderGroupIdentifier attribute from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.A0081
+      command: "readAttribute"
+      attribute: "AliroReaderGroupIdentifier"
+      response:
+          value: null
+
+    - label: "Step 10: TH reads AliroGroupResolvingKey attribute from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.F0e && DRLK.S.A0084
+      command: "readAttribute"
+      attribute: "AliroGroupResolvingKey"
+      response:
+          value: null
+
+    - label:
+          "Step 11a: TH sends SetAliroReaderConfig Command to DUT with the
+          following fields, SigningKey as signingkey, VerificationKey as
+          verificationkey, GroupIdentifier as groupidentifier, GroupResolvingKey
+          not present."
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && !DRLK.S.F0e && DRLK.S.C28.Rsp
+      command: "SetAliroReaderConfig"
+      arguments:
+          values:
+              - name: "SigningKey"
+                value: signingkey
+              - name: "VerificationKey"
+                value: verificationkey
+              - name: "GroupIdentifier"
+                value: groupidentifier
+              - name: "GroupResolvingKey"
+                value: ""
+
+    - label:
+          "Step 11b: TH sends SetAliroReaderConfig Command to DUT with the
+          following fields, SigningKey as signingkey, VerificationKey as
+          verificationkey, GroupIdentifier as groupidentifier, GroupResolvingKey
+          as groupresolvingkey."
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0e && DRLK.S.C28.Rsp
+      command: "SetAliroReaderConfig"
+      arguments:
+          values:
+              - name: "SigningKey"
+                value: signingkey
+              - name: "VerificationKey"
+                value: verificationkey
+              - name: "GroupIdentifier"
+                value: groupidentifier
+              - name: "GroupResolvingKey"
+                value: groupresolvingkey
+
+    - label:
+          "Step 12: TH sends SetUser Command to DUT to create an Aliro user with
+          the following values: OperationType as 0( DataOperationTypeEnum.Add),
+          UserIndex as 1, UserName as alirouser, UserUniqueID as 111, UserStatus
+          as 1(UserStatusEnum.OccupiedEnabled), UserType as
+          0(UserTypeEnum.UnrestrictedUser), CredentialRule as
+          0(CredentialRuleEnum.Single)"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F08 && DRLK.S.C1a.Rsp
+      command: "SetUser"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "OperationType"
+                value: 0
```

```suggestion
                value: DataOperationTypeEnum.Add
```
and similar elsewhere for credential types, user types, credential rules, and so on.

---

will be updated in python script